### PR TITLE
Add Apache Spark

### PR DESCRIPTION
**Pull request by zero323**
https://github.com/awesome-spark/awesome-spark

Apache Spark is a popular data processing framework and of the most active open source projects today.

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title. For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- [x] The entry in the Awesome list should include a short description about the list project/theme (not the list itself), and should be added at the bottom of the appropriate category.
-  The list I added
	- [x] **has been around for at least 30 days,**
	- [x] is the result of hard work and is the best I could possibly produce,
	- [x] is a non-generated Markdown file in a GitHub repo,
	- [x] is not a duplicate,
	- [x] includes a succinct description of the project/theme at the top ([example](https://github.com/willempienaar/awesome-quantified-self)),
	- [x] includes a project logo/illustration (whenever possible) at the top right of the readme ([example](https://github.com/sindresorhus/awesome-electron)). (The logo should link to the project website or any relevant website. The logo should be high-DPI, so set it to maximum half the width of the original image.),
	- [x] only has awesome items **with descriptions** *(Awesome lists are curations of the best, not of everything)*,
	- [x] contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- [x] has a Table of Contents section **named `Contents`** as the first section,
	- [x] has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)* *(If you use a license badge, it should be SVG, not PNG)*,
	- [x] has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- [x] doesn't include a Travis badge,
	- [x] has been checked for inconsistent formatting, spelling, and grammar:
		- [x] Each link description starts with an uppercase character and ends with a period.
		- [x] Drop all the `A` / `An` prefixes in the descriptions.
		- [x] Consistent naming. For example, `Node.js`, not `NodeJS` or `node.js`.

### CODE REVIEW

**Comment by zero323** [comment]
cc @eliasah

**Comment by eliasah** [comment]
LGTM

File: readme.md [hunk 1]
```diff
@@ -253,6 +253,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Hadoop](https://github.com/youngwookim/awesome-hadoop) - Framework for distributed storage and processing of very large data sets.
 - [Data Engineering](https://github.com/igorbarinov/awesome-data-engineering)
 - [Streaming](https://github.com/manuzhang/awesome-streaming)
+- [Apache Spark](https://github.com/awesome-spark/awesome-spark) - Apache Spark is an unified engine for large-scale data processing.
```
Comment (davisonio) [inline readme.md]: Another thing in the actual list is that it'd look better without spaces between a couple of the contents bullet points.

Comment (eliasah) [inline readme.md]: I get your first comment but I'm not sure that I understand the second one. Would you care giving more explanation please ?

Comment (zero323) [inline readme.md]: Gentle ping @sindresorhus @davisonio

**Comment by sindresorhus** [comment]
>  includes a succinct description of the project/theme at the top (example),

Not followed.

> The logo should be high-DPI, so set it to maximum half the width of the original image.

Not followed.

> If you use a license badge, it should be SVG, not PNG

Not followed.

**Comment by sindresorhus** [comment]
ping @zero323

**Comment by zero323** [comment]
I believe these are OK now @sindresorhus.

**Comment by sindresorhus** [comment]
These are not done:

> includes a succinct description of the project/theme at the top (example),
> Not followed.

Still not. I see no description of Apache Spark at the top of your readme.

> The logo should be high-DPI, so set it to maximum half the width of the original image.
> Not followed.

The image doesn't exist anymore.

**Comment by zero323** [comment]
I don't see much interest from other contributors, and personally I have other thing on my mind,  so I don't think we'll be pursuing this after all. Sorry.

**Comment by sindresorhus** [comment]
I think you're giving up too easily. It takes more work to interest contributors, and being on this list definitely helps that, but if you're not willing to put in that work, I totally understand :)

**Comment by oluies** [comment]
Updated Readme with an Apache Spark description, is the image OK? 
 @sindresorhus

**Comment by sindresorhus** [comment]
Yup. Added.

**Pull request closed without merging**